src/include/sof/lib/dma.h

```diff
@@ -541,8 +541,8 @@ int dma_copy_from_host_nowait(struct dma_copy *dc,
 			      int32_t size);
```

Comment:
typo in commit: " changes also prove.c" ->  "changes also probe.c"

Reply:
I think my new version, that asks for the correct alignment from the DMA driver should fix the issue in the first comment.  Forgot about the typo. I'll push the typo fix that later. I have it now in my repo.